src/pages/ConciergePage.js

```diff
@@ -15,7 +15,14 @@ const propTypes = {
 
         /** Currently logged in user email */
         email: PropTypes.string,
-    }).isRequired,
+    }),
+};
+
+const defaultProps = {
+    session: {
+        authToken: null,
+        email: null,
```

Comment:
the `session.email` is removed https://github.com/Expensify/App/pull/12157/files#diff-dee473482f17c430c462f98d9335ed3f4eea15f0e9ff8117a183f49117edebea

I think we can eliminate the `session.email`.